packages/app/src/background/services/credentials/index.ts

```diff
@@ -1,35 +1,41 @@
-import { EventName } from "@cryptkeeperzk/providers";
+import { EventName } from "@cryptkeeperzk/providers/dist/src/event";
```

Comment:
Same here with dist import. Check all the cases in this PR.

Reply:
I am running into some issues with this. When I just import `EventName` from `"@cryptkeeperzk/providers"`, Typescript cannot resolve the enum into its values, and so when I try to use `EventName.NEW_VERIFIABLE_CREDENTIAL`, it just returns `undefined`.

I tried exporting the enum as a `const`, but this violates the `isolatedModules` rule.

I also tried importing it from `@cryptkeeperzk/providers/src/event`, but then these files cannot be found in the actual build since they are outside the root dir of the package.

Do you have any suggestions on how to fix this?